zookeeper-server/src/main/java/org/apache/zookeeper/server/PrepRequestProcessor.java

```diff
@@ -1029,7 +1029,8 @@ private void validateCreateRequest(String path, CreateMode createMode, Request r
                     }
                 }
                 // If the znode path contains open read access node path prefix, add (world:anyone, r) in addition
-                if (ZNodeGroupAclUtil.getOpenReadAccessPathPrefixes().stream().anyMatch(path::contains)) {
+                ZNodeGroupAclProperties zNodeGroupAclProperties = ZNodeGroupAclProperties.getInstance();
```

Comment:
Minor: usually with Singletons, you can use `ZNodeGroupAclProperties.getInstance()` directly without having to assign it to a variable